.github/workflows/build-test.yml

```diff
@@ -75,6 +75,10 @@ jobs:
       - name: Install winget
         if: matrix.platform == 'windows-2022'
         uses: Cyberboss/install-winget@v1
+      - name: Add MinGW to PATH on Windows
+        if: matrix.platform == 'windows-2022'
+        run: echo "/c/mingw64/bin" >> $GITHUB_PATH
```

Comment:
From what I read on the issue, I don't believe this works (but I could be wrong)